client/a8c-for-agencies/sections/sites/site-preview-pane/style.scss

```
@@ -0,0 +1,118 @@
+.site-preview__pane {
+	display: flex;
+	flex-direction: column;
+
+	.preview-pane__navigation {
+		clip-path: inset(0 -1px -1px -1px);
+		box-shadow: 0 0 0 1px var(--studio-gray-0), 0 1px 2px var(--color-neutral-0);
+		margin: 0;
+
+		.section-nav-tab__link {
+			color: #000;
+		}
+
+		.section-nav-tab__link:hover {
+			color: #000;
+			background-color: inherit;
+		}
+
+		.section-nav-tabs__list {
+			padding: 0 32px;
+		}
+
+		.section-nav-tab__text {
+			font-weight: 500;
+		}
+
+		.section-nav-tab:not(.is-selected):hover {
+			border-bottom: none;
+		}
+
+		.section-nav-tab {
+			&.is-selected {
+				border-bottom-color: var(--color-neutral-70);
+			}
+		}
+
+		.section-nav-tabs__dropdown .select-dropdown__container {
+			max-width: revert;
+		}
+
+		.section-nav__panel:has(.is-dropdown) {
+			padding: 16px;
+		}
+
+		.select-dropdown.is-open {
+			height: revert;
+		}
+
+		@media (min-width: 481px) {
+			.section-nav-group {
+				.is-dropdown {
+					width: 100%;
+				}
+			}
+		}
+
+		@media (max-width: 480px) {
+			.section-nav-group {
+				margin-top: 0;
+			}
+			.section-nav-tabs__list {
+				margin: 0 24px;
+				padding: 0 !important;
+				border: 1px solid var(--color-neutral-10) !important;
+			}
+
+			.section-nav__mobile-header {
+				margin: 16px 24px 0 24px;
+				width: 88%;
+				border: 1px solid var(--color-neutral-10) !important;
+				border-radius: 2px;
+				justify-content: space-between;
+			}
+
+			.section-nav-tab__link {
+				color: var(--color-neutral-70);
+				font-weight: 400;
+			}
+
+			.section-nav-tab {
+				.section-nav-tab__link {
+					&:hover {
+						color: var(--color-accent);
+						background-color: var(--color-neutral-5);
+					}
+				}
+
+				&.is-selected {
+					.section-nav-tab__link {
+						color: #fff;
+
+						&:hover {
+							color: #fff;
+							background-color: var(--studio-automattic-60);
+						}
+					}
+				}
+			}
+		}
+	}
+}
+
+@media (max-width: 480px) {
+	.section-nav {
+		box-shadow: 0 0 0 1px #fff !important;
+	}
+
+	.section-nav.is-open {
+		.section-nav__panel {
+			border-top: none !important;//px solid white;
```

```suggestion
			border-top: none !important;
```

This comment was the same in the jetpack-cloud file as well, added by me by mistake in #88682, so probably better to remove it while we catch it.

---

yep, better to remove them